packages/cron/src/lib/metrics.js

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
+import { Histogram, Registry } from 'prom-client'
+
+/**
+ * @exports
+ * @typedef {Histogram<"byteLength">} RetrievalDurationSecondsMetric
+ */
+
+/**
+ * @param {Registry} registry
+ * @returns {RetrievalDurationSecondsMetric}
+ */
+export function createRetrievalDurationSecondsMetric(registry) {
```

Comment:
This should be milliseconds instead of seconds, unless I missed a sneaky conversion somewhere, milliseconds is the unit that is used everywhere else. Also, the conversion is handled automatically in grafana.

Reply:
from https://prometheus.io/docs/practices/naming/

> ...should use base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers). See below for a list of base units.

There was a conversion, but using the prom-client `Histogram` directly as a metric wouldn't have included very good type-checking so I added some types to help. https://github.com/nftstorage/nft.storage/pull/1945/commits/440e07e9885bf9544b59b99ac62aebc466ba7ec3#diff-5823c1dd943cae9a4760e898922c5e7a0344a231d51560db2541882f2a9beb41R12